ecr/main.tf

```diff
@@ -26,3 +26,20 @@ resource "aws_ecr_lifecycle_policy" "hpc-resource-provisioner-ecr-lifecycle-poli
   repository = aws_ecr_repository.hpc-resource-provisioner-ecr.name
   policy     = data.aws_ecr_lifecycle_policy_document.expire_images.json
 }
+
+# tfsec:ignore:aws-ecr-repository-customer-key
+resource "aws_ecr_repository" "workflow-svc-ecr" {
+  name = "bbp-workflow-svc"
```

Comment:
do we want to get rid off the bbb prefix?

Reply:
I wouldn't mind... @eleftherioszisis do you think this would have a big impact on the deployment?

Reply:
We can leave it for another PR. I saw we still use bb--worflow* for file names in the deployment repo.
Up to you :)

Reply:
Yeah, I will do that later at some point.

Reply:
After discussion with @eleftherioszisis , let's save it for a future PR
Don't merge yet though... I've seen some unexpected changes in `terraform plan` against production that I want to properly check first

Reply:
Domain changes confirmed harmless!